DATA/production/configurations/asyncReco/setenv_extra.sh

```diff
@@ -802,6 +802,10 @@ if [[ $ALIEN_JDL_THINAODS == "1" ]] ; then
   export ARGS_EXTRA_PROCESS_o2_aod_producer_workflow+=" --thin-tracks"
 fi
 
+if [[ $ALIEN_JDL_PREPROPAGATE == "1" ]] ; then
+  export ARGS_EXTRA_PROCESS_o2_aod_producer_workflow+=" --propagate-tracks 1 --propagate-tracks-max-xiu 5"
```

Comment:
The `propagate-tracks` is a boolean switch, so its presence in the options turns it `on` regardless of the following argument, better remove `1` to avoid confusion if somebody wants to modify this option.